students/mitek_michal/lesson_01_basics/first_number_after_decimal_point.py

```diff
@@ -0,0 +1,11 @@
+
+def print_first_number_after_decimal_point():
+
+    number = float(input("Provide a number "))
+
+    fractional_number = str(number - int(number))[1:]
```

Comment:
This is ok. You can also consider the solution like this: int ((number - int (number)) * 10) 
In my opinion, it is more clear and without any string slicing.